Calendar/CalLogHelpers/TimelineFunctions.ps1

```diff
@@ -67,7 +67,7 @@ function BuildTimeline {
     FindOrganizer($script:FirstLog)
 
     # Ignorable and items from Shared Calendars are not included in the TimeLine.
-    [array]$InterestingCalLogs = $script:EnhancedCalLogs | Where-Object { $_.LogType -eq "Core" -and $_.IsFromSharedCalendar -eq $False }
+    [array]$InterestingCalLogs = $script:EnhancedCalLogs | Where-Object { $_.LogType -eq "Core" -and $_.SharedFolderName -eq "Not Shared"  }
```

Comment:
Looks like code formatter is call out the double space before the closing bracket `}`